docs/standalone-payment-service-integration.md

```diff
@@ -34,48 +34,43 @@ InPlayer.Account.signUp({
 .then(data => console.log(data));
 ```
 
-You can find this method defined in detail [here](https://inplayer-org.github.io/inplayer.js/classes/Account.html#method-signUp). 
+You can find this method defined in detail [here](https://inplayer-js.netlify.app/classes/account.html#signup). 
 
-*When the registration is completed and delivers a successful response, among the response parameters you will find a unique authentication token. This token indicates that the respective end-user is authenticated. 
+When the registration is completed and delivers a successful response, among the response parameters you will find a unique authentication token. This token indicates that the respective end-user is authenticated.
 
-To authenticate or login an already existing account, you need to call the ‘InPlayer.Account.authenticate()' method.
+To authenticate or login an already existing account, you need to call the ‘InPlayer.Account.signIn()' method.
 
 
 Example:
 ```
-InPlayer.Account.authenticate({
- email: 'test@test.com',
- password: 'test123',
- clientId: '123-123-hf1hd1-12dhd1',
- referrer: 'http://localhost:3000/',
- refreshToken: '528b1b80-ddd1hj-4abc-gha3j-111111'
+InPlayer.Account.signIn({
+email: 'test@test.com',
+password: 'test123',
+clientId: '123-123-hf1hd1-12dhd1',
+referrer: 'http://localhost:3000/',
+refreshToken: '528b1b80-ddd1hj-4abc-gha3j-111111'
 })
 .then(data => console.log(data));
 ```
 
-Proceed [here](https://inplayer-org.github.io/inplayer.js/classes/Account.html#method-authenticate) to learn more about this method.
+Proceed [here](https://inplayer-js.netlify.app/classes/account.html#signin) to learn more about this method.
 
 
 On the other hand, if you decide to use our API, these are the endpoints you need:
 
 
-Start by firing the [POST] /accounts request and enter all the relevant details that will get your end-user registered in our system. 
+Start by firing the [POST] /v2/accounts/customers request and enter all the relevant details that will get your end-user registered in our system. 
 
 
 Example:
 ```
-curl https://services.inplayer.com/accounts \
--d full_name="John Doe" \
--d username=john@example.com \
--d password=foobar123 \
--d password_confirmation=foobar123 \
--d type=consumer \
--d grant_type=password \
--d client_id=507d866c-4e33-445d-b027-f163d0099f54 \
--d metadata[phone]=bar
+curl https://services.inplayer.com/v2/accounts/customers \
+-H 'authorization: Bearer Access-Token' \
+-H 'content-type: application/x-www-form-urlencoded' \
+-d email=john@example.com \
 ```
 
-For a more detailed description of this endpoint, refer to [this section](https://docs.inplayer.com/api/accounts/#operation/createAccount) of our API documentation.
+For a more detailed description of this endpoint refer to [this section](https://docs.inplayer.com/api/accounts/#operation/getSocial) of our API documentation.

```

Comment:
I think that `/getSocial` is the wrong endpoint. 

Perhaps it should be this one - https://docs.inplayer.com/api/accounts/#operation/createAccount